### PR TITLE
ADD documentation for P1D import

### PR DESCRIPTION
**Pull request by MiquelIR**
### Description:

> Documentation for the importation of tm profiles from P1d files.

### Tasks:

- [ ] Tasks to do
- [x] Tasks done

### Links:

> Add links for each file modified

* [telemesura #Càrrega de corbes de telemesura mitjançant fitxers](http://builds.gisce.net/powerp-docs/powerp_p1d_docu_to_telemeasure/ca/distri/telemedida/#carrega-de-corbes-de-telemesura-mitjancant-fitxers)
* [telemedida #Carga de curvas de telemedida mediante ficheros](http://builds.gisce.net/powerp-docs/powerp_p1d_docu_to_telemeasure/es/distri/telemedida/#carga-de-curvas-de-telemedida-mediante-ficheros)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_p1d_docu_to_telemeasure/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_p1d_docu_to_telemeasure/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_p1d_docu_to_telemeasure/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_p1d_docu_to_telemeasure/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/telemedida.md [hunk 1]
```diff
@@ -94,22 +94,26 @@ En aquesta pestanya trobem diferents camps configurables i informatius.
...
-és el de comprovar si el comptador referenciat dins el fitxer té o no perfils existents
-per el període entre les dates d'inici i final del fitxer. Si es troben perfils els eliminarà.
-El següent pas és crear tots els perfils nous.
+Aquest assistent suporta dos tipus de fitxers diferents:
+
+* Format **.curva** que s'extreu dels comptadors mitjançant els dispositius TPL.
+
+* Format [P1D](_static/telemedida/P1D.pdf) definit per la REE.
+
+A més a més es poden introduir un o altre d'aquests dos formats de fitxers massivament
```
Comment (jaumef) [inline docs/distri/telemedida.md]: Posa això amb un "tip":

```
!!! Tip "Consell"
    <text>
```

File: docs/distri/telemedida.md [hunk 2]
```diff
@@ -94,22 +94,26 @@ En aquesta pestanya trobem diferents camps configurables i informatius.
...
-Dins l'assistent i indicant la unitat d'energia que s'utilitza en el fitxer especificat ja
-està llest per polsar el botó **Carregar**. El primer pas que executa el procés de càrrega
-és el de comprovar si el comptador referenciat dins el fitxer té o no perfils existents
-per el període entre les dates d'inici i final del fitxer. Si es troben perfils els eliminarà.
-El següent pas és crear tots els perfils nous.
+Aquest assistent suporta dos tipus de fitxers diferents:
+
+* Format **.curva** que s'extreu dels comptadors mitjançant els dispositius TPL.
+
+* Format [P1D](_static/telemedida/P1D.pdf) definit per la REE.
```
Comment (jaumef) [inline docs/distri/telemedida.md]: Sempre que es treu algo d'un document oficial i públic, encara que facilitis un extracte, posa d'on surt i l'enllaç:

(definit per la REE...) "en la resolució/BOE XXXX de 201X pàgines XX-XX"

File: docs/distri/telemedida.md [hunk 3]
```diff
@@ -94,22 +94,26 @@ En aquesta pestanya trobem diferents camps configurables i informatius.
...
+
+A més a més es poden introduir un o altre d'aquests dos formats de fitxers massivament
+comprimits en un fitxer ".zip".
+
+Dins l'assistent ens caldrà seleccionar el fitxer que volem carregar **1** i el tipus de
```
Comment (jaumef) [inline docs/distri/telemedida.md]: Els números, ja que es refereixen a la figura, posa'ls entre parèntesis

**Comment by jaumef** [comment]
@MiquelIR Després d'això, fes el "merge.sh" i ja estarà :+1:

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_p1d_docu_to_telemeasure/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_p1d_docu_to_telemeasure/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**